src/lib/DepositRecordSerializer.js

```diff
@@ -32,7 +32,6 @@ import {
   VocabularyField,
 } from './fields';
 
-
```

Comment:
Minor: I guess I am guilty here as I have been aiming for there to be 2 new lines after imports and before the first class declaration like in Python. Is there a style guide I could follow? (I believe I tried installing prettier at some point, but didn't manage to make it work on my computer. Is it even the reference we use?)

Reply:
I am just using prettier with the default configuration. Let's have a talk about it and align :)